api_testing/apis/test_plan_api.py

```diff
@@ -363,4 +363,16 @@ def set_configuration(self):
         config_file = os.path.abspath('api_testing/config/test_plan.json')
         with open(config_file, "r") as read_file:
             payload = json.load(read_file)
-        super().set_configuration(payload=payload)
\ No newline at end of file
+        super().set_configuration(payload=payload)
+
+    def create_multiple_test_plan_with_same_article(self):
```

Comment:
This could take no_of_test_plans and returns a list of the created test plans.

Reply:
done